website/source/layouts/api.erb

```diff
@@ -22,6 +22,15 @@
           <li<%= sidebar_current("api-acl-policies") %>>
             <a href="/api/acl/policies.html">Policies</a>
           </li>
+          <li<%= sidebar_current("api-acl-roles") %>>
```

Comment:
I think we should spend a few minutes talking about the order of this list. The whole thing could probably be updated.